server/game/gameSystems/DistributeAmongTargetsSystem.ts

```diff
@@ -61,7 +61,7 @@ export abstract class DistributeAmongTargetsSystem<TContext extends AbilityConte
         const legalTargets = properties.selector.getAllLegalTargets(context);
 
         // auto-select if there's only one legal target and the player isn't allowed to choose 0 targets
-        if (!properties.canChooseNoTargets && legalTargets.length === 1) {
+        if ((!properties.canChooseNoTargets && !properties.optional) && legalTargets.length === 1) {
```

Comment:
If we make the change I suggested in the Vambrace Flamethrower file, we can still do this if we instead use `context.ability.optional`. Same for change on line 74